### PR TITLE
Update introduction

### PR DESCRIPTION
**Pull request by Miljar**
Remove the information about backwards compatibility during the lifetime of the current major version. It is too limiting.

### CODE REVIEW

File: src/introduction.apib [hunk 1]
```diff
@@ -1,6 +1,4 @@
-This is the new Teamleader API. We will make additions and changes to this API, but don't worry; we will keep the documented endpoints backwards compatible for the lifetime of this major API version.
-
-Do you have any feedback, do you miss functionality to support your use case? Let us know via [api@teamleader.eu](mailto:api@teamleader.eu).
+This is the new Teamleader API.  If you have any feedback, or are you missing functionality to support your use case? Let us know via [api@teamleader.eu](mailto:api@teamleader.eu).
```
Comment (Pegasuz) [inline src/introduction.apib]: Should we still call it "new"?

Comment (Pegasuz) [reply src/introduction.apib]: Maybe "latest" is more suitable ?

Comment (Miljar) [reply src/introduction.apib]: "This is the documentation of the Teamleader API." ?

Comment (tiagobastos) [reply src/introduction.apib]: *Teamleader Focus

Comment (tiagobastos) [inline src/introduction.apib]: nitpick
```suggestion
This is the documentation of the Teamleader Focus API.

If you have any feedback, or are you missing functionality to support your use case? Let us know via [api@teamleader.eu](mailto:api@teamleader.eu).
```

**Review by mrtus — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by Pegasuz — APPROVED** [review]

**Pull request merged**